installing/installing-preparing.adoc

```diff
@@ -155,7 +155,7 @@ endif::openshift-origin[]
 |xref:../installing/installing_gcp/installing-restricted-networks-gcp-installer-provisioned.adoc#installing-restricted-networks-gcp-installer-provisioned[X]
 |xref:../installing/installing_openstack/installing-openstack-installer-restricted.adoc#installing-openstack-installer-restricted[X]
 |xref:../installing/installing_rhv/installing-rhv-restricted-network.adoc#installing-rhv-restricted-network[X]
-|
+|xref:../installing/installing_bare_metal_ipi/ipi-install-installation-workflow.adoc#ipi-install-installation-workflow[X]
```

Comment:
My only thought is that the other link in the bare metal column goes to the [Overview page](https://deploy-preview-36922--osdocs.netlify.app/openshift-enterprise/latest/installing/installing_bare_metal_ipi/ipi-install-overview.html). Should this link go to the same page?

Reply:
I made the point to the documentation on installing IPI in Bare Metal, which contains a topic that guides you in creating a disconnected registry. I followed the same as the other documentation like for example: AWS.
The link to Default is correct.

Reply:
I would choose to have specific documentation for installation in a restricted network environment. However, as the product does not have specific documentation, I chose to indicate the documentation that addresses this supportability.